tests/integration/defs/test_e2e.py

```
@@ -2624,3 +2624,59 @@ def test_get_ci_container_port():
+                             model_subdir=model_subdir,
+                             streaming=False,
+                             use_pytorch_backend=True,
+                             use_mpirun=False,
+                             tp_size=1,
+                             concurrency=concurrency,
+                             num_requests=num_requests)
+
+        output = runner()
+        results[concurrency] = output
+
+    print(f"\n=== Benchmark Results Comparison ===")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)